### DISCUSSION TITLE
参考文献悬挂缩进

### DISCUSSION THREAD

**Discussion by sixswallow** [whutug/whu-thesis#258] [General]
如何设置参考文献中的悬挂缩进呢，目前模板的著者-年份格式采用的是悬挂缩进1字符，学院给的模板是缩进2字符的，想修改怎么办呢

**Comment by xkwxdyy** [comment]
什么学院，提供模板。

**Comment by sixswallow** [comment]
感谢您的及时回复。

是测绘学院的模板。
![Snipaste_2024-04-15_11-49-12](https://github.com/whutug/whu-thesis/assets/111326445/aeee079b-eaa8-4055-9ed6-d4ce5fef257e)

**Reply by xkwxdyy** [reply]
提供一下源文件吧，以及这是今年更新的吗？

**Reply by sixswallow** [reply]
不是今年更新的，但是学院给我们发的模板就是这个
[学术学位硕士论文撰写格式【2021】（仅供参考）.pdf](https://github.com/whutug/whu-thesis/files/14973816/2021.pdf)
